DATA/production/qc-async/trd.json

```diff
@@ -0,0 +1,151 @@
+      "DigitTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::trd::DigitsTask",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "dataSamplingPolicy",
+          "name": "trdall"
+        },
+       "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        }
+      },
+      "TrackletsTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::trd::TrackletsTask",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "dataSamplingPolicy",
+          "name": "trdall"
+        },
+        "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        }
+      },
+      "PulseHeight": {
+        "active": "true",
+        "className": "o2::quality_control_modules::trd::PulseHeight",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "dataSamplingPolicy",
+          "name": "trdall"
+        },
+        "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        }
+      }    
+    },
+    "checks": {
+      "QcCheck": {
+        "active": "false",
+        "className": "o2::quality_control_modules::trd::RawDataCheck",
+        "moduleName": "QcTRD",
+        "policy": "OnAny",
+        "detectorName": "TRD",
+        "dataSource": [
+          {
+          "type": "Task",
+          "name": "RawDataTask",
+          "MOs": ["trackletsperevent"]
+          }
+        ]
+      },
+      "PulseHeightCheck": {
+        "active": "false",
+        "className": "o2::quality_control_modules::trd::PulseHeightCheck",
+        "moduleName": "QcTRD",
+        "policy": "OnAny",
+        "detectorName": "TRD",
+        "dataSource": [
+          {
+          "type": "Task",
+          "name": "PulseHeightTask",
+          "MOs": ["trackletsperevent"]
+          }
+        ]
+      }
+    }
+  },
+  "dataSamplingPolicies": [
+    {
+      "id": "trdall",
+      "active": "true",
+      "machines": [],
+      "query" : "digits:TRD/DIGITS/0;tracklets:TRD/TRACKLETS/0;triggers:TRD/TRKTRGRD/0;rawstats:TRD/RAWSTATS/0",
+      "samplingConditions":[
+        {
+          "condition": "random",
+          "fraction": "0.9",
```

Comment:
if you need almost all data (90%), you could consider attaching QC directly to the data, as illustrated in here:
https://github.com/AliceO2Group/QualityControl/blob/175baf49916e92552349654a8ce9b3fdb414cd42/Framework/basic-no-sampling.json#L34